README.md

```diff
@@ -166,6 +167,7 @@ C++ is a high-performance language by itself, however, there are plenty of useca
 
 Algorithms is a fundamental topic indeed, usually language-independent, and such books age like a good wine - that's why following the last language Standard is not necessary here.
 
+* [A Complete Guide to Standard C++ Algorithms](https://leanpub.com/cpp-algorithms-guide) (Šimon Tóth, last updated on 2023)
```

Comment:
This book is a recent C++ algorithms open book by Simon Toth, a great C++ teacher I found on LinkedIn.
It can be freely downloaded. I read it and enjoyed every single example of it. It also covers C++20 features.